### PR TITLE
Update cb-setup.md

### PR DESCRIPTION
**Pull request by sbudhirajadoc**
remove information about role, license and profile because they aren't needed to set up code builder.

<!--- PR title should follow the pattern: <type>(optional scope): <description>.
please refer to the types and format here: https://www.conventionalcommits.org/en/v1.0.0/#summary

### What does this PR do?
updates the setup topic. 
### What issues does this PR fix or reference?
#<Insert GitHub Issue>, @W-14401079@

### Functionality Before
<insert gif and/or summary>

### Functionality After
<insert gif and/or summary>

### CODE REVIEW

File: docs/_articles/en/codebuilder/cb-setup.md [hunk 1]
```diff
@@ -40,14 +40,7 @@ From Setup, enter `Installed Packages` in the Quick Find box, then select **Inst
...
-
-The listed license and profile is required to use Code Builder.
-
-| **Role**    | **License**                                                                                                                                                                                                                 | **Profile**       |
-| ----------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------- |
-| Team Member | [Free Limited Access](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/dev_hub_license.htm) or [Identity](https://help.salesforce.com/s/articleView?id=sf.users_license_types_available.htm&type=5) | DX Limited Access |
-|             |                                                                                                                                                                                                                             |                   |
+Add any team members who aren’t already users in the Code Builder org.
```
Comment (smaddox-sf) [inline docs/_articles/en/codebuilder/cb-setup.md:43]: @sbudhirajadoc - Thank you for the quick change.  What license can be used has been a common question.  (I understand it was a common question for DevOps Center also.) I suggest we keep the info that Code Builder works with free licenses.  Maybe add a sentence after "Add any team members who aren’t already users in the Code Builder org." - along the lines of:

Code Builder works with the [Identity](https://help.salesforce.com/s/articleView?id=sf.users_license_types_available.htm&type=5) or the [Free Limited Access](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/dev_hub_license.htm) license.

**Review by smaddox-sf — APPROVED** [review]
I suggested an edit.  However, I'm approving because we could publish the change as-is (without the suggested edit) and then make the update in the next release.  I know deadline is tomorrow AM.

File: docs/_articles/en/codebuilder/cb-setup.md (hunk 1 above)
Comment (sbudhirajadoc) [reply docs/_articles/en/codebuilder/cb-setup.md]: done.

**Comment by sbudhirajadoc** [comment]
> I suggested an edit. However, I'm approving because we could publish the change as-is (without the suggested edit) and then make the update in the next release. I know deadline is tomorrow AM.

I've made your suggested update.

**Review by smaddox-sf — APPROVED** [review]

**Pull request merged**